scripts/helpers/new-chain-setup-mainnet.sh

```diff
@@ -13,3 +13,14 @@ dlc-link-eth add-signer 0xdf4d8B54dE476B674f1832B95984fFa7e223d47B # PierTwo
 dlc-link-eth set-attestor-gpk 'xpub6C1F2SwADP3TNajQjg2PaniEGpZLvWdMiFP8ChPjQBRWD1XUBeMdE4YkQYvnNhAYGoZKfcQbsRCefserB5DyJM7R9VR6ce6vLrXHVfeqyH3'
 
 dlc-link-eth set-btc-fee-recipient 021b34f36d8487ce3a7a6f0124f58854d561cb52077593d1e86973fac0fea1a8b1
+
+dlc-link-eth set-threshold 4
+
+# dlc-link-eth grant-role-on-manager DLC_ADMIN_ROLE $MEDIUM_MULTISIG
```

Comment:
Is this comment section intentional?

Reply:
unf yes, as these don't have their scripts written yet. And they happen so infrequently, it is not a priority now.

But I left a note here for our future selves, so we will know the steps required.